### DISCUSSION TITLE
Self host supabase - nginx

### DISCUSSION THREAD

**Discussion by Sulkar** [supabase/supabase#27307] [Questions]
How can I self-host Supabase? I have a domain and Docker is running. What settings do I need to configure in the .env file and in the server block of Nginx?

Do i need separate domains for:
- storage
- auth
- studio
- ...

Do you have a sample configuration?

**Comment by GaryAustin1** [comment]
I'll throw this out there as no one has replied yet.  I personally have no idea...
https://blog.activeno.de/the-ultimate-supabase-self-hosting-guide
https://www.blackslate.io/articles/how-to-install-self-host-supabase
